pyclam/manifold.py

```diff
@@ -424,7 +464,7 @@ def __iter__(self) -> Iterable[Cluster]:
     def __str__(self) -> str:
         # Cashing value because sort can be expensive on many clusters.
         if 'str' not in self.cache:
-            self.cache['str'] = ','.join(sorted(list(map(str, self.clusters))))
+            self.cache['str'] = ', '.join(sorted(list(map(str, self.clusters))))
```

Comment:
I'm becoming a big fan of the `::` separator, I submit that for your consideration here

Reply:
meh. I was looking at clusters in the debugger and the space helped visually.